biosimulations/libs/shared/ui/src/lib/tab-page/tab-page-tab.component.html

```diff
@@ -4,7 +4,7 @@
     {{ heading }}
   </ng-template>
 
-  <div [ngClass]="{'partial-width': partialWidth}">
+  <div [ngClass]="{'partial-width': partialWidth}" style="height: 100%">
```

Comment:
This change is not possible. This is a common library across multiple apps.  We would need to check the other apps as well before making the change. 
@jonrkarr  could you approve/disaprove?